app/tests/integration/nimbus/conftest.py

```diff
@@ -448,4 +455,47 @@ def fixture_experiment_default_data():
                 "featureValue": "",
             }
         ],
+        "populationPercent": "100",
+        "totalEnrolledClients": 55,
     }
+
+@pytest.fixture(name="check_ping_for_experiment")
+def fixture_check_ping_for_experiment():
+    def _check_ping_for_experiment(exp=None):
+        for item in requests.get("http://ping-server:5000/pings").json():
+            if "experiments" in item["environment"]:
+                for key in item["environment"]["experiments"]:
+                    if exp in key:
+                        return True
+                    else:
+                        continue
+                else:
+                    return False
```

Comment:
Ah good ol `for...else...` Yep looks good.  This is totally fine but just some suggestions to avoid so much indentation.

```py
data = requests.get("http://ping-server:5000/pings").json()
experiments_data = [item for item in data if 'experiments' in item['environment']][0]
return exp in experiments_data.keys()
```
experiments_data